src/gui/editors/PianoRoll.cpp

```diff
@@ -247,6 +248,15 @@ PianoRoll::PianoRoll() :
 	connect( markChordAction, &QAction::triggered, [this](){ markSemiTone(SemiToneMarkerAction::MarkCurrentChord); });
 	connect( unmarkAllAction, &QAction::triggered, [this](){ markSemiTone(SemiToneMarkerAction::UnmarkAll); });
 	connect( copyAllNotesAction, &QAction::triggered, [this](){ markSemiTone(SemiToneMarkerAction::CopyAllNotesOnKey); });
+	connect(ConfigManager::inst(), &ConfigManager::valueChanged,
+		[this](QString const& cls, QString const& attribute, QString const& value)
+		{
+			if (cls != "midi" && attribute != "autoquantize")
```

Comment:
Logic bug; should be OR(~A,~B), or NAND(A,B), but not AND(~A,~B).
Condition is false if cls=="midi", even it attribute doesn't match, and vice versa, so we skip over the early return more often than intended.

Reply:
Fixed with 8c977c9